node/manifest.test.ts

```diff
@@ -433,22 +433,7 @@ test('Generates a manifest with layers', () => {
   expect(manifest2.layers).toEqual(layers)
 })
 
-test('Throws an error if the regular expression contains a negative lookahead and support for the PCRE engine is disabled', () => {
-  const functions = [{ name: 'func-1', path: '/path/to/func-1.ts' }]
-  const declarations = [{ function: 'func-1', pattern: '^\\/((?!api|_next\\/static|_next\\/image|favicon.ico).*)$' }]
-
-  expect(() =>
-    generateManifest({
-      bundles: [],
-      declarations,
-      functions,
-    }),
-  ).toThrowError(
-    /^Could not parse path declaration of function 'func-1': Regular expressions with lookaheads are not supported$/,
-  )
-})
-
-test('Accepts regular expressions with lookaheads if support for the PCRE engine is enabled', () => {
+test('Accepts regular expressions with lookaheads', () => {
```

Comment:
This test still passes the flag

Reply:
Oops! Removed in https://github.com/netlify/edge-bundler/pull/580/commits/6ff20a3084bccbb108376c715811e0a31722cb3e.